core/src/main/resources/error/error-classes.json

```diff
@@ -426,6 +426,11 @@
     ],
     "sqlState" : "42000"
   },
+  "DATA_SOURCE_NOT_FOUND" : {
+    "message" : [
+      "Failed to find data source: <provider>. Please find packages at https://spark.apache.org/third-party-projects.html"
```

Comment:
hmmm.... add single quotation for the html url?

Reply:
Just added single quotation around the URL, thanks!